File: ncDados/dVendasNfe.vb
Namespace nsVenda

    Public Class ColecaodVendasNfe
        Inherits List(Of dVendasNfe)
    End Class

    Public Class dVendasNfe
        Private _cupom As String
        Private _dataVenda As String
        Private _valor As String
        Public Property Cupom() As String
            Get
                Return _cupom
            End Get
            Set(ByVal value As String)
                _cupom = value
            End Set
        End Property
        Public Property DataVenda() As String
            Get
                Return _dataVenda
            End Get
            Set(ByVal value As String)
                _dataVenda = value
            End Set
        End Property
        Public Property Valor() As String
            Get
                Return _valor
            End Get
            Set(ByVal value As String)
                _valor = value
            End Set
        End Property

    End Class
End Namespace

File: Modelos/dVendasNfe.cs
using System;
using System.Collections.Generic;

namespace Modelos
{
    public class ColecaodVendasNfe : List<dVendasNfe>
    {
    }

    public class dVendasNfe
    {
        private string _cupom;
        private string _dataVenda;
        private string _valor;

        public string Cupom
        {
            get => _cupom;
            set => _cupom = value;
        }

        public string DataVenda
        {
            get => _dataVenda;
            set => _dataVenda = value;
        }

        public string Valor
        {
            get => _valor;
            set => _valor = value;
        }
    }

}
